maxi-blocks/maxi-blocks
# Issue 3958 - Background position after reset

# Description



Fixes #3958 

# How Has This Been Tested?

1. Add different maxi blocks (text-maxi, container-maxi, etc)
2. Add a background layer and select set seperate
3. Change one of the position fields and then reset
4. Change it again and it should still update on the screen
5. Test reset on other fields including responsive.



# Test checklist



**_ Front/Back Testing _**


**_ Pre-Code Testing _**


# Checklist
